src/main/webapp/scripts/script.js

```diff
@@ -82,6 +82,38 @@ function parseTime(time){
   return timeString;
 }
 
+/**
+ * Function to prove check if business is still opened if it hasn't been updated
+ * @param {Object} saleCard - Salecard object that contains its info
+ * @return {boolean} 
+ */
+function isOpened(saleCard) {
+  if (saleCard.isTemporarilyClosed) {
+    return false;
+  }
+
+  const businessStartTime = saleCard.startTime;
+  const businessEndTime = saleCard.endTime;
+
+  const startTime = new Date();
+  startTime.setHours(businessStartTime.hour, businessStartTime.minute, businessStartTime.second);
```

Comment:
Just a question about setHours(), do you think we can implement this method to get rid of `my parseTime()` function ?